### PR TITLE
Added managedBy,cloudPcGroupDisplayName,gracePeriodInHours,localAdmin…

### PR DESCRIPTION
**Pull request by youche-microsoft**
Roadmap id: 14288

### CODE REVIEW

File: api-reference/beta/resources/cloudpcprovisioningpolicy.md [hunk 1]
```diff
@@ -40,6 +40,20 @@ Represents a Cloud PC provisioning policy.
 |microsoftManagedDesktop|[microsoftManagedDesktop](../resources/microsoftManagedDesktop.md)|The specific settings for the Microsoft Managed Desktop, which enables customers to get a managed device experience for the Cloud PC. Before you can enable Microsoft Managed Desktop, an admin must configure it.|
 |onPremisesConnectionId|String|The ID of the cloudPcOnPremisesConnection. To ensure that Cloud PCs have network connectivity and that they domain join, choose a connection with a virtual network that’s validated by the Cloud PC service.|
 |windowsSettings|[cloudPcWindowsSettings](../resources/cloudpcwindowssettings.md)|Specific Windows settings to configure while creating Cloud PCs for this provisioning policy.|
+|managedBy|[cloudPcManagementService](#cloudpcmanagementservice-values)|Specifies which services manage the Azure network connection. Possible values are: `windows365`, `devBox` and `unknownFutureValue`. Read-only.
```
Comment (RuiHou105) [inline api-reference/beta/resources/cloudpcprovisioningpolicy.md:43]: You don't need to define the enum `cloudPcManagementService `again in line 50, as actually you reuse the enum resource [here ](https://github.com/microsoftgraph/microsoft-graph-docs/pull/15792/files#diff-e5720e91111f17527fbeb78ecf90164278efa64ede434328fd79f46cca833b57)right? So please remove line 50 - 56 and update this type to `[microsoft.graph.cloudPcManagementService](../resources/cloudpconpremisesconnection.md#cloudpcmanagementservice-values)`

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
@youche-microsoft  please help resolve comments in this PR. Thanks

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 2]
```diff
@@ -2046,6 +2046,56 @@
             "CreatedDateTime": "2020-11-06T15:42:07.177Z",
             "WorkloadArea": "Devices and apps",
             "SubArea": "Cloud PC"
+        },
+        {
+            "ChangeList": [
+              {
+                "Id": "04b7a9f2-66ab-4ba4-8732-8f65c5f8a966",
```
Comment (RuiHou105) [inline changelog/Microsoft.CloudManagedDesktop.json:2053]: In the changelog, these Ids should be the same. According to your graph schema PR, the generated changloe uses this one a896f747-29fd-4ca3-8dc8-15056e9dc255. So please use this one for the six Ids here.

File: api-reference/beta/resources/cloudpcprovisioningpolicy.md (hunk 1 above)
Comment (youche-microsoft) [reply api-reference/beta/resources/cloudpcprovisioningpolicy.md]: Fixed

File: changelog/Microsoft.CloudManagedDesktop.json (hunk 2 above)
Comment (youche-microsoft) [reply changelog/Microsoft.CloudManagedDesktop.json:2053]: Fixed

**Review by RuiHou105 — APPROVED** [review]

**Comment by RuiHou105** [comment]
@JarbasHorst @Lauragra , please help review this PR when you got time. Thanks!

File: api-reference/beta/api/cloudpcprovisioningpolicy-get.md [hunk 3]
```diff
@@ -214,6 +215,62 @@ Content-Type: application/json
...
+
+# [HTTP](#tab/http)
+<!-- {
+  "blockType": "request",
+  "name": "get_cloudpcprovisioningpolicy_3"
+}
+-->
+
+``` http
+GET https://graph.microsoft.com/beta/deviceManagement/virtualEndpoint/provisioningPolicies/{id}?$select=id,description,displayName,displayName,domainJoinConfiguration,imageDisplayName,imageId,imageType,onPremisesConnectionId,windowsSettings,managedBy,cloudPcGroupDisplayName,gracePeriodInHours,localAdminEnabled,alternateResourceUrl
```
Comment (JarbasHorst) [inline api-reference/beta/api/cloudpcprovisioningpolicy-get.md:236]: For clarity, please replace the placeholder "{id}" with a real-looking ID.

File: api-reference/beta/api/cloudpcprovisioningpolicy-get.md [hunk 4]
```diff
@@ -214,6 +215,62 @@ Content-Type: application/json
...
+  "truncated": true,
+  "@odata.type": "microsoft.graph.cloudPcProvisioningPolicy"
+}
+-->
+
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
```
Comment (JarbasHorst) [inline api-reference/beta/api/cloudpcprovisioningpolicy-get.md:253]: For clarity, please provide real-looking values in this response body. Examples with real-looking values in the, request URL, requests, and response bodies help customers understand the API better. That's also the consistency bar for Microsoft Graph API docs.

**Comment by JarbasHorst** [comment]
Hello @youche-microsoft, I reviewed your PR and left some comments. Please, take a look when you will have time.

Also, to avoid merge conflicts, please merge PR [17103](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17103) into your branch to include my fixes and before addressing my new comments.

File: api-reference/beta/resources/cloudpcprovisioningpolicy.md [hunk 5]
```diff
@@ -81,6 +86,11 @@ The following is a JSON representation of the resource.
   "onPremisesConnectionId": "String",
   "windowsSettings": {
     "@odata.type": "microsoft.graph.cloudPcWindowsSettings"
-  }
+  },
```
[the remark this answers is not in the document]
Comment (youche-microsoft) [reply api-reference/beta/resources/cloudpcprovisioningpolicy.md]: Fixed, thanks!

File: api-reference/beta/resources/cloudpcprovisioningpolicy.md [hunk 6]
```diff
@@ -40,6 +40,11 @@ Represents a Cloud PC provisioning policy.
 |microsoftManagedDesktop|[microsoftManagedDesktop](../resources/microsoftManagedDesktop.md)|The specific settings for the Microsoft Managed Desktop, which enables customers to get a managed device experience for the Cloud PC. Before you can enable Microsoft Managed Desktop, an admin must configure it.|
 |onPremisesConnectionId|String|The ID of the cloudPcOnPremisesConnection. To ensure that Cloud PCs have network connectivity and that they domain join, choose a connection with a virtual network that’s validated by the Cloud PC service.|
 |windowsSettings|[cloudPcWindowsSettings](../resources/cloudpcwindowssettings.md)|Specific Windows settings to configure while creating Cloud PCs for this provisioning policy.|
+|managedBy|[microsoft.graph.cloudPcManagementService](../resources/cloudpconpremisesconnection.md#cloudpcmanagementservice-values)|Specifies which services manage the Azure network connection. Possible values are: `windows365`, `devBox` and `unknownFutureValue`. Read-only.
```
[the remark this answers is not in the document]
Comment (youche-microsoft) [reply api-reference/beta/resources/cloudpcprovisioningpolicy.md]: Fixed, thanks!

File: api-reference/beta/api/cloudpcprovisioningpolicy-get.md (hunk 4 above)
Comment (youche-microsoft) [reply api-reference/beta/api/cloudpcprovisioningpolicy-get.md:263]: I've made some of the values more real-looking. Could you please take a look to see if they require further changes?

**Review by AshleyYangSZ — APPROVED** [review]

File: api-reference/beta/api/cloudpcprovisioningpolicy-get.md (hunk 4 above)
Comment (JarbasHorst) [reply api-reference/beta/api/cloudpcprovisioningpolicy-get.md:263]: Look great, thanks :)

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review. Please, merge PR [17116](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17116) into this branch to include my final edits.

**Comment by AshleyYangSZ** [comment]
> @Lauragra this PR is ready for final review. Please, merge PR [17116](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17116) into this branch to include my final edits.

A reminder the corresponding schema will go alive at around Jun 9th. This PR couldn't be merged before that.

**Review by Lauragra — APPROVED** [review]

**Comment by RuiHou105** [comment]
@Lauragra  As the graph schema already went alive last week, let's merge this one. Thanks :)

**Pull request merged**